### ISSUE TITLE
Windows 7:  xjadeorc location

### ISSUE THREAD

**Issue by Dynamosity** [x42/xjadeo#17]
On Windows 7, where should I put the xjadeorc file? 
I have it in C:\Users\Michael\AppData\Roaming but it doesn't seem to be changing anything in xjadeo player.
Help appreciated as this is not clear from the docs.

**Comment by x42** [comment]
`$HOMEDRIVE$HOMEPATH\xjadeorc` would be `C:\Users\Michael\xjadeorc`  a text file (no .txt extesion)

**Comment by Dynamosity** [comment]
Thanks Robin. That works :-)) I tried to configure the splash screen off and it worked.

There are some options I don't understand in xjadeorc which I need to configure.

 **use '-v -m -1' to list midi-ports** 
In that list will the first port be 0, second 1 etc?

 and **'xjadeo -V'** lists available drivers. 

Do I do those in the xjadeorc file ? Or somewhere else? I am running Windows 7 x64.

Many thanks for your help so far.

**Comment by x42** [comment]
> In that list will the first port be 0, second 1 etc?

yes. Alternatively you can leave it as default (-1) and use the portmidi setup application.
See "PmDefaults Application" at http://portmedia.sourceforge.net/portmidi/
